Sources/CGFloatExtensions.swift

```diff
@@ -14,13 +14,8 @@ extension CGFloat {
     public var center: CGFloat { return (self / 2) }
 
     /// EZSwiftExtensions
-    public func toRadians() -> CGFloat {
```

Comment:
@Khalian could you please mark it with deprecated and renamed to `degreesToRadians` as done in many other cases through `ArrayExtensions` and others...
As we move forward we will drop such methods, but give people a bit time to see that it will be removed soon.